README.md

```
@@ -35,6 +35,16 @@ pytest -v
-# 특정 파일의 특정 함수만 테스트 
+# 특정 파일의 특정 함수만 테스트
 pytest {테스트파일명}.py -k {테스트함수명}
-```
\ No newline at end of file
+
+# 특정 폴더의 파일들만 테스트
+pytest blog/tests/admin -q
+```
+
+cronjob 실행 등록
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)